src/app/container/container.component.html

```diff
@@ -79,16 +79,16 @@ <h3 class="pull-right">
     <h3>
       <div class="pull-right my-entry-button-group">
         <div class="btn-group" role="group">
-          <button id="publishToolButton" type="button" [ngClass]="{'disabled': publishDisable()}" class="btn btn-warning" (click)="publish()" [(ngModel)]="published"
-            [disabled]="!thisisValid" btnCheckbox>
+          <button id="publishToolButton" mat-raised-button color="secondary" (click)="publish()" [(ngModel)]="published"
+            [disabled]="publishDisable()" btnCheckbox [matTooltipShowDelay]="1000" [matTooltipPosition]="'before'" matTooltip="Publish/unpublish the tool">
```

Comment:
To quote Gary from PRs of mine he's reviewed, how many times is `publishDisable()` called? Could you just have it as a property instead?

I realize this is probably beyond the scope of what this PR is intended for, but mentioning anyway.

Reply:
This should be a pretty quick fix so I'll do it in this PR.